doc/lib.rst

```diff
@@ -88,10 +91,10 @@ Algorithms
 ----------
 
 * `algorithm <algorithm.html>`_
-  This module implements some common generic algorithms like sort or binary search.
+  This module implements some common generic algorithms on `openArray`s.
```

Comment:
How is that description better? `sort` is something I need to do, openArray an implementation detail.